### PR TITLE
Remove service worker

### PR DESCRIPTION
**Pull request by ryota-murakami**
**Status**
- [ ] WIP
- [x] Ready for review
- [ ] Needs testing

**Deploy after merge (delete what needn't be deployed)**
- hyperion (frontend)

**Related issues (delete if you don't know of any)**
Closes https://github.com/withspectrum/spectrum/issues/3519

Remove offline-plugin and Unregister ServiceWerker along by CRA doc https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#opting-out-of-caching

thanks😀

### CODE REVIEW

**Comment by brianlovin** [comment]
Not totally opposed to doing this cc @mxstbr

**Comment by brianlovin** [comment]
Not sure about the ci failure @ryota-murakami - going to restart the job and see if it was a cypress fluke

**Comment by brianlovin** [comment]
And we'll get back to you about whether we should merge this and totally remove the service worker. I'm not opposed; this + adding the ZEIT CDN for our logged-out users might be a fine-enough balance of perf and stability

**Comment by mxstbr** [comment]
This will break web push notifications—we don't want to remove the sw entirely, we only want to remove the caching part! /cc @NekR

**Comment by ryota-murakami** [comment]
@mxstbr I see, i just submit ServiceWorker register code from unregister .

**Comment by ryota-murakami** [comment]
On my local, SW is activated and running.
screen shot 2018-07-20 at 19 48 03

File: config-overrides.js [hunk 1]
```diff
@@ -90,57 +87,7 @@ module.exports = function override(config, env) {
...
-            // location will be a thing
-            // eslint-disable-next-line no-restricted-globals
-            return new URL('./index.html', location);
-          },
-          requestTypes: ['navigate'],
-        },
-      ],
-      rewrites: arg => arg,
-      ServiceWorker: {
-        entry: './public/push-sw.js',
```
Comment (ryota-murakami) [inline config-overrides.js:134]: I think offline-plugin was register SW here.

File: src/registerServiceWorker.js [hunk 2]
```diff
@@ -15,18 +15,12 @@ export type ServiceWorkerResult = {
...
-  ? `${process.env.PUBLIC_URL}/service-worker.js`
-  : `${process.env.PUBLIC_URL}/push-sw-v1.js`;
-
 export default function register(): Promise<ServiceWorkerResult> {
   if ('serviceWorker' in navigator) {
     return new Promise(res => {
       window.addEventListener('load', () => {
         navigator.serviceWorker
-          .register(swUrl)
+          .register('push-sw.js')
```
Comment (ryota-murakami) [inline src/registerServiceWorker.js:23]: Drop offline-plugin, then I replaced SW register here.

File: config-overrides.js (hunk 1 above)
Comment (ryota-murakami) [reply config-overrides.js:134]: I think offline-plugin was register SW here.

**Comment by ryota-murakami** [comment]
@mxstbr I know your super busy guy, if you have a little spare time, could you please review about,

-[ ] Is my commits only remove OfflinePlugin?
-[ ] Is my commits still work service worker matter function(push notification etc.)

Thank you so much😀

**Pull request closed without merging**